src/frontend/apps/lti_site/components/LiveInfoBar/index.spec.tsx

```diff
@@ -66,10 +112,145 @@ describe('<LiveInfoBar />', () => {
   });
 
   it('renders with invalid startDate', () => {
-    render(<LiveInfoBar title={'title'} startDate={'some date'} />);
+    const mockedVideo = videoMockFactory({
+      allow_recording: false,
+      title: 'title',
+    });
+
+    render(
+      wrapInVideo(
+        <LiveInfoBar startDate={'some date'} />,
+        mockedVideo,
+      ),
+    );
 
     screen.getByRole('heading', { name: 'title' });
     expect(screen.queryByText('some date')).not.toBeInTheDocument();
     expect(screen.queryByText('Invalid DateTime')).not.toBeInTheDocument();
   });
+
+  it('allows you to edit the title', async () => {
+    const mockedVideo = videoMockFactory({
+      allow_recording: false,
+      title: 'title',
+    });
+
+    render(
+      wrapInVideo(
+        <LiveInfoBar startDate={'some date'} />,
+        mockedVideo,
+      ),
+    );
+
+    let $titleInput = screen.getByRole('textbox', { name: 'Enter title of your live here' });
+    $titleInput.focus();
+    await userEvent.clear($titleInput);
+    await userEvent.type($titleInput, 'new title');
+    $titleInput.blur();
+    $titleInput = screen.getByRole('textbox', { name: 'Enter title of your live here' });
```

Comment:
```suggestion
```

There is no need to query it again. `titleValue` already contains the modification normally